core/process/aru.php

```diff
@@ -654,7 +654,11 @@
 			$resultPkms = $mysqli->query($req);
 
 			while ($resultPkms && $dataPkm = $resultPkms->fetch_object()) {
+				$dataPkm->today = $locales->TODAY;
```

Comment:
Probably this should go as a property of the main response, but as it is an array maybe in per trainer and not per pokemon.